open-api.yaml

```diff
@@ -1850,6 +1910,66 @@ paths:
         - $ref: '#/components/parameters/Content-Type'
     parameters:
       - $ref: '#/components/parameters/indexUid'
+  '/indexes/{indexUid}/documents/delete':
+    post:
+      operationId: indexes.documents.remove
+      summary: Delete documents
+      description: Delete a selection of documents based on a filter.
+      tags:
+        - Documents
+      security:
+        - apiKey: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                offset:
+                  type: number
+                  description: Number of documents to skip.
+                  default: 0
+                limit:
+                  type: number
+                  description: Maximum number of documents returned.
+                  default: 20
+                fields:
+                  type: array
+                  description: 'Array of attributes whose fields will be present in the returned documents. By default all attributes will be returned.'
+                  items:
+                    type: string
+                    example: '["title", "overview"]'
+                    default: '["*"]'
```

Comment:
these should be removed, no ?

Reply:
Oh yes you’re right, does this fix your comments? https://github.com/meilisearch/specifications/pull/236/commits/98fbdcb02bd1dd7febf913d1ab637af27d58c8d7

Reply:
yes perfect 👌

Reply:
Huum I don’t know; maybe @gmourier will have an idea?

Reply:
What do you mean by generating API def @ahmednfwela?

Reply:
this comment here ? https://github.com/meilisearch/specifications/pull/236#issuecomment-1534856855

Reply:
It's built for every commit changing the openapi.yml file